cmake/Omim.cmake

```diff
@@ -0,0 +1,139 @@
+# Function for setting target platform:
```

Comment:
Is it possible to rename this file to Omim-helpers.cmake or to Omim-routines.cmake?

Reply:
I suggest OmimRoutines, according to cmake default module names:
https://cmake.org/cmake/help/v3.0/manual/cmake-modules.7.html

Reply:
OmimHelpers.cmake sgtm

Reply:
@mpimenov done.